Source/WTF/wtf/RetainPtr.h

```diff
@@ -164,6 +164,16 @@ template<typename T> class RetainPtr {
 
     static constexpr PtrType hashTableDeletedValue() { return reinterpret_cast<PtrType>(-1); }
 
+    template<typename U>
+    friend class RetainPtrGetterRetains;
+
+    template<typename U = T>
+    std::enable_if_t<!std::is_convertible_v<U, id>, __unsafe_unretained PtrType*> internalValuePointer() WARN_UNUSED_RETURN {
+        static_assert(std::is_same_v<T, U>, "explicit specialization not allowed");
+        RELEASE_ASSERT(!m_ptr);
+        return reinterpret_cast<PtrType*>(&m_ptr);
```

Comment:
Does `&fromStorageType(m_ptr)` work here instead of `reinterpret_cast<PtrType*>(&m_ptr)`?

The goal is to get rid of the `const`-ness of `CFTypeRef` as defined in `CFBase.h`:

```
typedef const CF_BRIDGED_TYPE(id) void * CFTypeRef;
```

Reply:
`&fromStorageType(m_ptr)` does not compile.